src/helpers/createCalendar.ts

```diff
@@ -1,104 +1,71 @@
-import { days, daysInMonth } from "./fixtures";
-import {CalendarMatrix} from '../ts-generalTypes/InitialStateInterfaces'
+import { daysInMonth } from "./fixtures";
+import { DayType } from "../ts-generalTypes/propTypes";
 
 export let date: Date = new Date();
```

Comment:
what is this?
I see that you also using same variable inside of createCalendarMatrix function. You shouldn't use variable with same naming in one file

Reply:
and in general I believe that you don't need this variable